env_docker.go

```diff
@@ -356,6 +348,37 @@ func (d *dockerRunnable) IsRunning() bool {
 	return d.usedNetworkName != ""
 }
 
+func (d *dockerRunnable) RunOneOff(ctx context.Context) (output string, err error) {
+	if d.IsRunning() {
+		return "", errors.Errorf("%v is running. Stop or kill it first to restart.", d.Name())
+	}
+
+	i, ok := d.concreteType.(identificable)
+	if !ok {
+		return "", errors.Errorf("concrete type has at least embed runnable or future runnable instance provided by Runnable builder, got %T; not implementing identificable", d.concreteType)
+	}
+	if i.id() != d.id() {
+		return "", errors.Errorf("concrete type has at least embed runnable or future runnable instance provided by Runnable builder, got %T; id %v, expected %v", d.concreteType, i.id(), d.id())
+	}
+
+	d.logger.Log("Starting", d.Name())
+
+	// Make sure the image is available locally; if not wait for it to download.
+	if err = d.prePullImage(ctx); err != nil {
+		return "", err
+	}
+
+	cmd := d.env.execContext(ctx, "docker", append([]string{"run", "-t", "--rm"}, d.env.buildDockerRunArgs(d.name, d.ports, d.opts)...)...)
```

Comment:
Would this result in double `--rm` flags as we add one in [here](https://github.com/efficientgo/e2e/pull/28/files#diff-ab6d0ab6f93748074333740158654203b796c2ccb2379d33060072557287ff34R234)?

Reply:
uuupd